OpenEdXMobile/res/menu/my_courses.xml

```diff
@@ -3,7 +3,7 @@
     xmlns:app="http://schemas.android.com/apk/res-auto">
     <item
         android:id="@+id/menu_item_search"
-        app:showAsAction="ifRoom"
-        android:icon="@drawable/abc_ic_search_api_material"
-        android:title="@string/share_certificate_button" />
+        android:icon="@drawable/search_icon"
+        android:title="@string/search_for_courses"
```

Comment:
This is a wrong title, in this case, we can just use the `Discover` as the title.